src/main/kotlin/net/atos/zac/productaanvraag/ProductaanvraagService.kt

```diff
@@ -162,8 +162,8 @@ class ProductaanvraagService @Inject constructor(
 
     /**
      * Adds all betrokkenen which are present in the provided productaanvraag to the zaak for the set
-     * of provided (role types)[Betrokkene.RolOmschrijvingGeneriek] but only for those role types which are defined
-     * in the zaaktype of the specified zaak.
+     * of provided (role types)[Betrokkene.RoltypeOmschrijving] or [Betrokkene.roltypeOmschrijving] but only for those
```

Comment:
Is this correct? RoltypeOmschrijving or roltypeOmschrijving?

Reply:
I think it would be good to document the logic more clearly here. It's not very clear now I think.

```
als een productaanvraag-betrokkene uitsluitend een rolOmschrijvingGeneriek veld specificeert dan wordt dat gebruikt

als een productaanvraag-betrokkene uitsluitend een rolOmschrijving veld specificeert dan wordt dat gebruikt

als een productaanvraag-betrokkene zowel een rolOmschrijvingGeneriek als een rolOmschrijving veld specificeert dan wordt het rolOmschrijvingveld gebruikt (omdat deze meer specifiek is)
```

Reply:
Added the text above